src/System.CommandLine/Parsing/ValueResultExtensions.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
```

Comment:
Consider adding this code to ArgumentResult

Also, be sure the ValueResultOutcome method is used